app/src/main/java/com/framgia/soundclound/widget/IsSwipeViewPager.java

```diff
@@ -0,0 +1,55 @@
+package com.framgia.soundclound.widget;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.support.v4.view.ViewPager;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+
+import com.framgia.soundclound.R;
+
+/**
+ * Created by Sony on 1/4/2018.
+ */
+
+public class IsSwipeViewPager extends ViewPager {
+    private boolean isSwipe;
+
+    public IsSwipeViewPager(Context context) {
+        super(context);
+    }
+
+    public IsSwipeViewPager(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        TypedArray a = context.getTheme().obtainStyledAttributes(
+                attrs,
+                R.styleable.IsSwipeViewPager,
+                0, 0);
+        try {
+            isSwipe = a.getBoolean(R.styleable.IsSwipeViewPager_isSwipe, false);
+
+        } finally {
+            a.recycle();
+        }
+    }
+
+    private void init() {
+        isSwipe = true;
+    }
+
+    @Override
+    public boolean onInterceptTouchEvent(MotionEvent event) {
+        return isSwipe && super.onInterceptTouchEvent(event);
+    }
+
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
```

Comment:
WARNING: [ClickableViewAccessibility] Custom view `IsSwipeViewPager` overrides `onTouchEvent` but not `performClick`
 If a `View` that overrides `onTouchEvent` or uses an `OnTouchListener` does not also implement `performClick` and call it when clicks are detected, the `View` may not handle accessibility actions properly. Logic handling the click actions should ideally be placed in `View#performClick` as some accessibility services invoke `performClick` when a click action should occur.